### ISSUE TITLE
Crash with Exception: Cannot add part named 'Base'. It already exists.

### ISSUE THREAD

**Issue by lbcsy** [OrchardCMS/OrchardCore#13751]
Steps to reproduce:

1. Create a Content Type named "Base", "Base" show up in the Content Type list
2. Create a Content Part named "Base", but "Base" not show up in the Content Part list
3. Create a Content Part again with name "Base", exception raises

**Comment by hishamco** [comment]
> Create a Content Part named "Base", the creation has no error, but "Base" does not show up in the Content Part list.

Yep, it will not show up because each content type has a default content part with the same name in your case "Base", but if you are trying to add "Base1" it should appear in the parts list

I think we should display a warning message afterward

**Comment by lbcsy** [comment]
Adding a warning definitely helps the user to understand what is going on. Btw, the vice versa, i.e., it should not allow an user to add a Content Type with a name of an existing part's name.

**Comment by hishamco** [comment]
Sure, @lbcsy do you want to push a PR for this?
